HISTORY.rst

```diff
@@ -9,6 +9,12 @@ Contributors to this version: Trevor James Smith (:user:`Zeitsperre`), Pascal Bo
 Bug fixes
 ^^^^^^^^^
 * ``build_indicator_module_from_yaml`` now accepts a ``reload`` argument. When re-building a module that already exists,  ``reload=True`` removes all previous indicator before creating the new ones. (:issue:`1192`,:pull:`1284`).
+* The test for french translations of official indicators was fixed and translations for CFFWIS indices, FFDI, KDBI, DF and Jetstream metric woollings have been added or fixed. (:pull:`1271`).
+
+New indicators
+^^^^^^^^^^^^^^
+* New indices and indicators for converting from snow water equivalent to snow depth (``snow_depth_from_amount``) and snow depth to snow water equivalent (``snow_amount_from_depth``) using snow density [kg/m^3]. (:pull:`1271`).
+* New indices and indicators for determining upwards radiation (CF variables `rsus` and `rlus`) from net and downwards longwave radiation (shortwave: `rss` and `rsds`; longwave: `rls` and `rlds`). (:pull:`1271`).
```

Comment:
```suggestion
* New indices and indicators for determining upwelling radiation (CF variables `rsus` and `rlus`) from net and downwelling longwave radiation (shortwave: `rss` and `rsds`; longwave: `rls` and `rlds`). (:pull:`1271`).
```
Sorry, but AFAIU `downwelling` (the word used by CF) better conveys the meaning of "coming from above". Whereas `downwards` could be understood as "positive down". What does @RondeauG  think about this vocab?

Reply:
I agree. We should stick as close as possible to CF nomenclature!

Reply:
Sounds good. I'll update the names of the functions themselves as well.